src/main/java/com/ita/if103java/ims/dto/ErrorInfoDto.java

```diff
@@ -0,0 +1,27 @@
+package com.ita.if103java.ims.dto;
+
+public class ErrorInfoDto {
+    private int status;
```

Comment:
Probably useless, since it's included in response by default

Reply:
Purpose of these changes is to return information  about an error in JSON format